Assets/Scripts/Layers/LayerTypes/WMSLayerGameObject.cs

```diff
@@ -0,0 +1,67 @@
+using Netherlands3D.Twin.Layers.Properties;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Netherlands3D.Twin.Layers
+{
+    /// <summary>
+    /// Extention of LayerGameObject that injects a 'streaming' dataprovider WMSTileDataLayer
+    /// </summary>
+    public class WMSLayerGameObject : CartesianTileLayerGameObject, ILayerWithPropertyData
+    {
+        private WMSTileDataLayer wmsProjectionLayer;
+        public WMSTileDataLayer WMSProjectionLayer
+        {
+            get
+            {
+                if (wmsProjectionLayer == null)
+                    wmsProjectionLayer = GetComponent<WMSTileDataLayer>();
```

Comment:
Je doet dit al in Awake, is dit niet overkill cq. als deze getriggered wordt voor Awake dan is er een ander probleem? :D

Reply:
nee want gek genoeg bleek de ref toch null te worden na instantiation